### PR TITLE
fix(ngx-login-client): upgrade ngx-login-client and provide value for WIT_API_PROXY

### PR DESCRIPTION
**Pull request by rohitkrai03**
Fixes https://github.com/fabric8-services/fabric8-auth/issues/680

- Version upgrade `ngx-login-client`
- Update response type of `sendEmailVerificationLink`. `Response` - > `HttpResponse`
- Explicitly provide value for `InjectionToken` `WIT_API_PROXY`

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @rohitkrai03 for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin.

File: src/app/app.module.ts [hunk 1]
```diff
@@ -179,6 +182,7 @@ export type StoreType = {
     ApiLocatorService,
     AreaService,
     authApiUrlProvider,
+    witApiProxyProvider,
```
Comment (joshuawilson) [inline src/app/app.module.ts]: This list is alphabetical, please move it down.

Comment (rohitkrai03) [reply src/app/app.module.ts]: Done

**Comment by joshuawilson** [comment]
How is the test coverage and does this change effect any tests?

**Comment by rohitkrai03** [comment]
@joshuawilson  I've reordered the providers in app.module alphabetically. Take a look.

**Comment by joshuawilson** [comment]
[test]

**Pull request merged**